src/style.css

```
@@ -0,0 +1,39 @@
+/* stylelint-disable no-descending-specificity */
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-flow: column nowrap;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars__star,
+.stars--0 .stars__star,
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:last-child {
```

delete all this classes, keep only stars__star
```suggestion
.stars__star {
...
```